app/ide-desktop/lib/dashboard/src/configurations/inputBindings.ts

```diff
@@ -101,4 +103,16 @@ export const BINDINGS = inputBindings.defineBindings({
     bindings: ['Mod+Shift+PointerMain'],
     rebindable: false,
   },
+  goBack: {
+    name: 'Go Back',
+    bindings: ['Mod+ArrowLeft', 'Mod+['],
```

Comment:
why `Mod+ArrowLeft`?

Reply:
that's alt+arrowleft though. and we probably want to use the browser's behavior rather than hijack it

Reply:
oh, on both windows and linux it's alt+arrowleft